alchemist/alchemist-cognitive-agents/src/main/kotlin/it/unibo/alchemist/model/implementations/actions/steeringstrategies/SinglePrevalent.kt

```diff
@@ -40,12 +40,12 @@ import it.unibo.alchemist.model.interfaces.geometry.euclidean2d.ConvexPolygon
  * oscillatory movements (this also known as shaking behavior).
  *
  * @param T concentration type
- * @param M type of nodes of the environment's graph.
+ * @param N type of nodes of the environment's graph.
  */
-class SinglePrevalent<T, M : ConvexPolygon>(
-    environment: Euclidean2DEnvironmentWithGraph<*, T, M, *>,
+class SinglePrevalent<T, N : ConvexPolygon>(
+    environment: Euclidean2DEnvironmentWithGraph<*, T, N, *>,
     pedestrian: Pedestrian2D<T>,
-    private val prevalent: List<SteeringAction<T, Euclidean2DPosition>>.() -> EuclideanNavigationAction<T, *, *, M, *>,
+    private val prevalent: List<SteeringAction<T, Euclidean2DPosition>>.() -> NavigationAction2D<T, *, *, N, *>,
```

Comment:
Projected type aliases are allowed, but I can't find a suitable name for the typealias of `NavigationAction2D<T, *, *, N, *>` - I think it can remain as it is. Ok for the type alias on steering actions.

Reply:
Since it's private, it can be just a shortener...
`private typealias NaviAction = NavigationAction2D<T, *, *, N, *>`

Reply:
Meh, it only appears once. Leave it be